docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: PROCEDURE
+:myfunctionone: hermetic_builds
+
+[id="hermetic-builds_{context}"]
+= Enabling hermetic builds
+
+Hermetic builds refer to a controlled build process in which a build system knows all the required resources and dependencies. This dependency visibility ensures that the build system can accurately collect, inquire, analyze, and manage all dependencies during the build process.
+
+In {ProductName}, hermetic builds are achieved by blocking network access to the build process. Therefore, you must pre-fetch all dependencies before the build can start.
+
+IMPORTANT: Hermetic builds disable network access, so the builds that require dependencies outside of its Git repository could fail. 
+
+Hermetic builds also focus on establishing a build pipeline that captures and maintains a precise record of the build's origin, dependencies, and modifications.
```

Comment:
```suggestion
Hermetic builds refer to a controlled build process in which a build system knows all the required resources and dependencies. This enhanced dependency visibility allows the build system to capture and produce a more precise record of the build's origin, dependencies, and modifications than it would otherwise be able to.

In {ProductName}, hermetic builds are achieved by blocking network access to the build process. Therefore, you must pre-fetch all dependencies before the build can start.

IMPORTANT: Hermetic builds disable network access, so the builds that require dependencies outside of its Git repository could fail. 
```

Reply:
Just stylistic. The last sentence seemed to be partly duplicating stuff from the first paragraph. I tried to incorporate it into the first.

Take it or leave it. Not critical. :)